includes/sensei-functions.php

```diff
@@ -282,3 +282,20 @@ function sensei_does_theme_support_templates() {
 
 	return in_array( $current_theme, $themes, true ) || current_theme_supports( 'sensei' );
 }
+
+/**
+ * Sorts lessons on edit.php?type=lesson default page
+ * @param type $query
+ */
+function sensei_admin_sort_lessons( $query ) {
+  if( !is_admin() )
```

Comment:
This function should follow the [coding standards for proper spacing](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#space-usage).